### ISSUE TITLE
Failed to compile - bug in keyframe_mapper.cpp

### ISSUE THREAD

**Issue by jfstepha** [CCNYRoboticsLab/ccny_rgbd_tools#13]
I had a problem compiling today:

```
ccny_rgbd/src/apps/keyframe_mapper.cpp:485:7: error: ‘result’ was not declared in this scope
```

Looks like "result < 0" just needs to be changed to "result_pcd < 0".

**Comment by jfstepha** [comment]
Sorry, I see this is fixed already in e990a7dba65968656a49288861e6e3ba6e564911.  I don't know why my pull didn't pick that up.

**Comment by ubuntuslave** [comment]
Try pulling again.

A small error we just fixed.

Carlos

On Mar 11, 2013, at 9:29 PM, Jon Stephan <notifications@github.com<mailto:notifications@github.com>> wrote:

I had a problem compiling today:

ccny_rgbd/src/apps/keyframe_mapper.cpp:485:7: error: ‘result’ was not declared in this scope

Looks like "result < 0" just needs to be changed to "result_pcd < 0".
